README.md

```diff
@@ -164,8 +164,16 @@ statements.
 Running the Proofs
 ------------------
 
-If Isabelle is set up correctly, a full test for the proofs in this repository
-can be run with the command
+In order to open the AInvs session, the file `Kernel_Config.thy` must be generated.
+This can be done by first setting the environment variable `L4V_ARCH` to `RISCV64`
```

Comment:
I don't get it. This is the "running the proofs" section. Can't `run_tests` handle building AInvs from a fresh repo checkout? What does `RISCV64` have to do with anything?
It seems you are trying to say that opening AInvs in Isabelle/JEdit won't work until `Kernel_Config.thy` is generated for the platform you're trying to load, and building ASpec is one way of triggering that generation.
These instructions aren't going to be very useful as-is for someone confused about why AInvs isn't working, and might be somewhat confusing if they aren't working on RISCV64.

Reply:
@lsf37 what's the real trigger for `Kernel_Config.thy` by the way?

Reply:
`Kernel_Config.thy` is generated by the target `c-config` in `spec/Makefile`. `design-spec` depends on that, and `ASpec` depends on `design-spec`.

Reply:
Then if we're saying what the things to invoke `make` on are, would `c-kernel` and `c-config` be the right targets then, rather than ASpec/CKernel?

Reply:
That would be the minimal command, but I'd keep the description to the proof sessions, because we might add more dependencies to them, but we are unlike to change the names of the main sessions.